app/move/views/view.njk

```diff
@@ -60,12 +60,50 @@
   </header>
 {% endblock %}
 
+{% set urls = {
+  personal_details: '/personal-details',
+  move_details: '/move-details',
+  court: '/court-information',
+  risk: '/risk-information',
+  health: '/health-information'
+} %}
+
+{% macro updateLink(category, categoryHeading) %}
+  {% if canAccess("move:update") %}
+    {% if not categoryHeading %}
+      {% set categoryKey = "moves::assessment.categories." + category + ".heading" %}
+      {% set stepsKey = "moves::steps." + category + ".heading" %}
+      {% set categoryHeading = t([categoryKey, stepsKey]) %}
+    {% endif %}
+    {% set updateLinkText = t("moves::assessment.categories.update_link", {category: categoryHeading}) %}
+    <p class="update-link"><a href="/move/{{ move.id }}{{ urls[category] }}" data-update-link="{{ category }}">{{ updateLinkText | safe }}</span></a></p>
+  {% endif %}
+{% endmacro %}
+
 {% block contentMain %}
+<style>
+.app-section {
```

Comment:
It would be good to move these to the sass rather than having inline CSS

Reply:
Absolutely. But which bit of SASS? Where should the macro in here go? Should it even be a section?

Reply:
`/common/assets/scss/core` or `/common/assets/scss/utilities` or even [`/common/assets/scss/overrides/_positioning`](https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/blob/master/common/assets/scss/overrides/_positioning.scss)? 

It feels like it would be better to have an override or helper class with what this is doing rather than just a classname of "section". Essentially all you need it to apply `position: relative` to the element so something like `.position-relative` or `.app-!-position-relative` (if using overrides) would be much clearer.

The same could be created for the link too. I'm not really a fan of tying classnames to their content - it makes the style too specific and confusing if the classname is ever re-used for something else. The classname for styles should be more generic description of what it is applying rather than describing what content is inside.

Reply:
Added `.app-!-position-relative` and `.app-!-position-top-right`